pinot-common/src/main/java/org/apache/pinot/pql/parsers/pql2/ast/HavingAstNode.java

```diff
@@ -31,4 +32,8 @@ public void updateBrokerRequest(BrokerRequest brokerRequest) {
     PredicateAstNode predicateAstNode = (PredicateAstNode) getChildren().get(0);
     RequestUtils.generateFilterFromTree(predicateAstNode.buildHavingQueryTree(), brokerRequest);
   }
+  @Override
+  public void updatePinotQuery(PinotQuery pinotQuery) {
+    // Skip HavingAstNode for now.
```

Comment:
Maybe throw exception to avoid undefined behavior?

Reply:
Will clean up Having code in another PR. Created an issue to track it. https://github.com/apache/incubator-pinot/issues/4309